pages/api-details.md

```diff
@@ -11,6 +11,6 @@ permalink: /details/
 
 _This site is a pilot project.  If you have feedback or questions, [let us know](https://github.com/18F/ED-Developer-Hub/issues)!_
 
-
+{% include swagger-section.html %}
```

Comment:
This is where the swagger UI interface will be placed, so content can appear above and below it.